uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-report.html

```diff
@@ -4,10 +4,10 @@
       <div class="chr_leave-report__heading panel-heading">
         <div class="row">
           <div class="col-sm-9">
-            <a href="#" class="chr_leave-report__export-button">
+            <!-- <a href="#" class="chr_leave-report__export-button">
```

Comment:
Please remove the markup altogether, if we want it back we can always use git history